torchaudio_contrib/functional.py

```diff
@@ -185,63 +185,61 @@ def magphase(complex_tensor, power=1.):
     return mag, phase
 
 
-def phase_vocoder(spect, rate, phi_advance):
+def phase_vocoder(complex_specgrams, rate, phase_advance):
     """
     Phase vocoder. Given a STFT tensor, speed up in time
     without modifying pitch by a factor of `rate`.
 
     Args:
-        spect (Tensor): (batch, channel, num_bins, time, complex=2)
-        rate (float): Speed-up factor
-        phi_advance (Tensor): Expected phase advance in each bin. (num_bins, 1)
+        complex_specgrams (Tensor):
+            (batch, channel, num_bins, time, complex=2)
+        rate (float): Speed-up factor, `rate` != 1.
```

Comment:
maybe making it even clearer what it means by `rate != 1`? like, it would raise an error? or nothing happens but we waste computation power? actually, it'd be nice if we print some warning when the rate is 1.

Reply:
Yeah I only included `rate != 1` so that the dimension in the doc made sense (i.e. new output time is `time//rate + 1`). The way I had it, the `TimeStretch` module checks if the rate is `1.0`, not calling  `phase_vocoder` if that's the case ([here](https://github.com/keunwoochoi/torchaudio-contrib/blob/80240574e428db8aa2f1551a3b3e1c9f1063786b/torchaudio_contrib/layers.py#L251)). We could put the warning there? I feel like the functional shouldn't check `rate == 1.0`.

Reply:
Hmm.. ok, because it changes the size, probably users wouldn't overlook when `rate==1.0`. I think then it'd be fine without warning or sth.